examples/config-tinygo/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	spinconfig "github.com/fermyon/spin/sdk/go/config"
```

Comment:
I'm torn between prefixing `spin*` before packages everywhere, or just where it clashes with standard library packages.
(nothing to change here, just wondering what your thoughts are)

Reply:
I think only where it clashes.